MessageBird/Objects/Request.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Text;
+
+namespace MessageBird.Objects
+{
+    /**
+     * Structure used for request signature verification via MessageBird.Utilities.RequestSigner
```

Comment:
Should now be `MessageBird.RequestSigner`

Reply:
Oops